Patches/Alpha Animals/ThingDefs_Races/AlphaAnimals_CE_Patch_Race_Overgrown_Colossus.xml

```diff
@@ -2,65 +2,67 @@
 <Patch>
 	<Operation Class="PatchOperationFindMod">
 		<mods>
-		<li>Alpha Animals</li>
+			<li>Alpha Animals</li>
 		</mods>

```

Comment:
Maybe move all colossi patches into one file to optimize things? They got same stats anyway.

Reply:
Mycoid Colossus doesn't have the same stats as an overgrown colossus, so separate files probably needed